purchase_sale_stock_inter_company/models/purchase_order.py

```diff
@@ -12,6 +12,9 @@ class PurchaseOrder(models.Model):
     def _prepare_sale_order_data(
         self, name, partner, dest_company, direct_delivery_address
     ):
+        """
+        Add additional fields when the purchase_stock module is installed.
+        """
```

Comment:
Docstring are not really this way, but:

```suggestion
        """Add additional fields when the purchase_stock module is installed.
        """
```

but anyway, please don't put docstring, but comment, as this is an override, and thus, it will replace the original docstring.

And please check the rest of the changes for fixing the docstring, or replacing it by a comment.

Reply:
Done. Please let me know if it’s OK for you now.